src/es_sfgtools/processing/assets/tiledb_temp.py

```diff
@@ -0,0 +1,188 @@
+import tiledb
+import numpy as np
+from pathlib import Path
+from datetime import datetime
+import pandas as pd
+from pandera import check_types
+from pandera.typing import DataFrame
+
+from typing import Optional,Dict
+
+from .observables import AcousticDataFrame,GNSSDataFrame,PositionDataFrame,ShotDataFrame
+
+filters = tiledb.FilterList([tiledb.ZstdFilter(5)])
+TimeDomain = tiledb.Dim(name="time", domain=(0,), tile=100, dtype="datetime64['ms']")
```

Comment:
I believe we ended up converting all timestamps to int64 because of concerns about differences in different software languages implementation of datetime.  It is safer and more performant to store as an int64 of unix milliseconds ie 
tiledb.Dim(
                name="time",
                domain=(0, 4102444800000),
                tile=86400000,
                dtype=np.int64,
                filters=tiledb.FilterList(
            [tiledb.DoubleDeltaFilter(), tiledb.ZstdFilter(level=7)],
            )
also the tile size is a somewhat confusing but important (tile is in the name of tiledb) setting, it has to do with how tiledb chunks and retrieves data, you should read the docs on that a bit but I think we want something like hourly or daily tiles, I am not sure what you have here would do exactly.